.gitignore

```diff
@@ -11,3 +11,6 @@ packages
 
 # Include when developing application packages.
 pubspec.lock
+
+.pub
```

Comment:
@grumpy nvm, I'll handle it. a couple other formatting changes that I want to do.